Logical div grouping in the about section

koksyncho opened the pull request:
Grouped the divs containing the title and description of the separate topics into the same div cells. This way they can be formatted more easily in the future.

koksyncho committed:
Logically grouped divs in the about section 

Grouped the Titles and Descriptions in the same divs in the about section so that they are grouped logically and so that they could be made responsive easier afterwards.

koksyncho committed:
Logically grouped divs in the about section

Grouped the Titles and Descriptions in the same divs in the about section so that they are grouped logically and so that they could be made responsive easier afterwards.

atodorov commented on index.html:
```diff
@@ -57,7 +57,7 @@ <h2 class="top-subheader">test case management system</h2>
             </div>
         </header>
-        <!-- About Section -->
+         <!-- About Section -->
```
Don't add extra space

atodorov commented on index.html:
```diff
@@ -68,58 +68,75 @@ <h2 class="top-subheader">test case management system</h2>
-                        <div class="rTableHead">Reporting</div>
-                        <div class="rTableHead">Integration</div>
+            <div class="div-table secondary">
+                <div class="div-table-row">
```
I think we don't need to have div elements for rows. Otherwise you are duplicating a table element structure but using different tags. Instead specify only the <div class="div-table-col-cell"> elements and use CSS to specify their width, possibly height, padding and alignment. If you do this correctly all of the 4 cels on each row should stack one next to the other and visually look like a table.

atodorov commented on style/style.css:
```diff
@@ -165,23 +165,31 @@ ul.topnav li [name="q"] {
-.rTableHead, .rTableFoot{
+.div-table-col-head {
     font-weight: bold;
+    padding: 0.3rem/*3px*/ 1.8%;
```
remove comments in CSS

atodorov commented on style/style.css:
```diff
@@ -165,23 +165,31 @@ ul.topnav li [name="q"] {
+    overflow: hidden;
+}
+.div-table-col-subhead {
+    padding: 0.3rem/*3px*/ 1.8%;
```
remove comment here as well

atodorov commented on style/style.css:
```diff
@@ -165,23 +165,31 @@ ul.topnav li [name="q"] {
+.div-table-col-cell {
     float: left;
     overflow: hidden;
     padding: 0.3rem/*3px*/ 1.8%;
```
here as well

atodorov commented on style/style.css:
```diff
@@ -165,23 +165,31 @@ ul.topnav li [name="q"] {
     width: 21%;
 }
-.rTable:after {
+.div-table:after {
```
Why do we need this? I don't understand what it does.

atodorov reviewed (changes requested):
You are getting there, some changes required though.

koksyncho committed:
Simplified take on the div "table"

Replaced the cell-like styling with the col-1 to col-12 separation we made earlier.  
Cleaned some comments and empty spaces.

koksyncho commented:
Thanks for the remarks, and sorry for answering so late.
I will close this pull request and make a new one with the requested changes.

atodorov commented:
@koksyncho FYI when you push again a branch for which there is an opened PR the PR will be updated with the latest changes. No need to close it and open a new PR for the changes.
